rllib/algorithms/ppo/tests/test_ppo_learner.py

```diff
@@ -117,6 +118,52 @@ def test_loss(self):
 
             check(learner_group_loss, policy_loss)
 
+    def test_save_load_state(self):
```

Comment:
add a short description on what this is testing. It's testing the save/load weights (not states) on learner groups